src/amo/components/AddonMoreInfo/index.js

```diff
@@ -164,6 +180,13 @@ export class AddonMoreInfoBase extends React.Component {
   }
 }
 
+export const mapStateToProps = (state) => {
+  return {
+    userId: state.user ? state.user.id : null,
```

Comment:
We could also pass the `isAddonAuthor` (bool) prop instead of computing this value in the `render()` method. The `isAddonAuthor()` function could be a selector living into `reducers/user` (like `isAuthenticated()`). This will make refactoring of the user info a bit easier if needed.

That being said, no need to change it if you don't want to :)